Costume/298_Azurill.yaml

```diff
@@ -0,0 +1,21 @@
+species: Azurill
+imagename: Hatenna
+displayname: Hatenna
+setname: Costume
+tags: [costume]
+item: [Psychic Gem]
```

Comment:
Gems are not in PBR...

Reply:
Again, closest alternative is Twisted Spoon, although that also isn’t used up. Unless any berries can do something similar, you’re stuck with that.